test/Test/Pos/CborSpec.hs

```diff
@@ -302,7 +303,7 @@ testAgainstFile name x expected =
               Right actual -> x `shouldBe` actual
 
 spec :: Spec
-spec = describe "Cbor.Bi instances" $ do
+spec = giveTestsConsts $ describe "Cbor.Bi instances" $ do
```

Comment:
Actually I think it's better to use test constants only in modules where running time depends on `k`.